book/src/hello_world.md

```diff
@@ -0,0 +1,22 @@
+# Hello World
+
+Let's write [a minimal VM](https://github.com/powdr-labs/powdr/blob/main/test_data/asm/simple_sum.asm) and generate a SNARK!
```

Comment:
This linked example is different from the included one, is that on purpose?

Reply:
oops!!